docs/02-guides/display-wysiwyg-content.mdx

```diff
@@ -0,0 +1,344 @@
+---
+title: Display WYSIWYG content
+description:
+  Let your users write their content without needing any HTML or React knowledge
+  while still being able to deliver a qualitative UI (performant, responsive,
+  accessible…) to customers.
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import ContactLink from "@site/src/components/ContactLink";
+import SinceVersion from "@site/src/components/SinceVersion";
+
+<SinceVersion tag="3.4" />
+
+<p>{frontMatter.description}</p>
+
+WYSIWYG stands for **W**hat **Y**ou **S**ee **I**s **W**hat **Y**ou **G**et. It
+means that your back office users write their content without needing any HTML
+or React knowledge. This is the case in most CMS tools nowadays. The output
+though is usually HTML which does not necessarily match the React components
+you've built in your Front-Commerce application.
+
+This is why we've built a `theme/modules/Wysiwyg` component in Front-Commerce.
+In this guide you will learn how to use it and how to customize its display.
+
+## `<Wysiwyg />` usage
+
+Here is an example of using this component:
+
+```tsx
+import React from "react";
+import Wysiwyg from "theme/modules/Wysiwyg";
+
+export default (props) => <Wysiwyg content={props.cms.contentWysiwyg} />;
```

Comment:
Bad example, components should preferably be named, this helps to avoid anonymous in react dev tools, as well as with HMR

```suggestion
export default function BlogContent(props) {
    return <Wysiwyg content={props.cms.contentWysiwyg} />;
}
```